microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -19,11 +19,11 @@ WebView2 is available for the following frameworks (or platforms) and programmin
    * [WPF](/dotnet/api/microsoft.web.webview2.wpf)
    * [Windows Forms](/dotnet/api/microsoft.web.webview2.winforms)
 
-* WinRT for WinUI 2 (UWP)
-   * [Core/C#](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
-   * [COM Interop/C++](/microsoft-edge/webview2/reference/winrt/interop/index)
+* WinRT for WinUI 2 (UWP) 
+   * [Core](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
```

Comment:
@david-risney - remove C# from this link text in this context?  Want to make this page as clear as possible re: standardized terminology useful in regular platform-specific articles.

Reply:
Yes this looks good to me.